src/components/Curriculum.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <section class="screen-box">
+    <header>
+      <span
+        class="screen-close"
+        @click="onClickClose"
+      />
+    </header>
+    <main>
+      <swiper :options="swiperOption">
+        <swiper-slide class="curriculum-overview">
+          <video-player
+            class="player"
+            :video-url="curriculum.videoUrl"
+            :thumbnail-url="curriculum.thumbnailUrl" 
+          />
+          <p class="title">
+            {{ curriculum.title }}
+          </p>
+          <p class="received-date">
+            <label>{{ curriculum.receivedDate }}</label>
+            <label>{{ curriculum.coachName }}</label>
+          </p>
+          <div class="description">{{ curriculum.description }}</div>
+          <p class="training-time">{{ `練習時間のめやす：${curriculum.trainingHours}時間` }}</p>
+        </swiper-slide>
+        <curriculum-step
+          v-for="(step, index) in curriculum.steps"
+          :key="index"
+          :data="step"
+          :order="index+1"
+        />
+      </swiper>
+    </main>
+  </section>
+</template>
+
+<script>
+import { swiper, swiperSlide } from 'vue-awesome-swiper'
+import VideoPlayer from '@/components/VideoPlayer.vue'
+import CurriculumStep from '@/components/CurriculumStep.vue'
+
+import 'swiper/dist/css/swiper.css'
+
+export default {
+  name: 'Curriculum',
+  components: {
+    swiper,
+    swiperSlide,
+    VideoPlayer,
+    CurriculumStep
+  },
+  data () {
+    return {
+      curriculum: {},
+      swiperOption: {
+        direction: 'vertical',
+        slidesPerView: 'auto',
+        spaceBetween: 30,
+        height: window.innerHeight
+      }
+    }
+  },
+  computed: {
+    recursionStep () {
+      const stepNum = this.curriculum.steps.length
+      if (stepNum === 0) {
+        return []
+      }
+      return this.decorateStep(stepNum-1)
+    }
+  },
+  methods: {
+    decorateStep (stepNum, index=0) {
+      var result = {
+        videoUrl: this.curriculum.steps[index].videoUrl,
```

Comment:
`this.curriculum.steps[index]` should be define as a variable to access its property easier